pass-data-client/pom.xml

```diff
@@ -4,16 +4,18 @@
   <parent>
     <groupId>org.eclipse.pass</groupId>
     <artifactId>pass-support</artifactId>
-    <version>0.3.0-SNAPSHOT</version>
```

Comment:
These version bumps _should_ happen during the release process, so something must have gone wrong last release

Reply:
Added a ticket to look into the automations https://github.com/eclipse-pass/main/issues/501